tests/functional/adapter/materialization/test_view_security.py

```diff
@@ -22,6 +30,15 @@ def project_config_update(self):
             "models": {
                 "+materialized": "view",
                 "+view_security": "invoker"
+            },
+            "seeds": {
```

Comment:
Move data types tests to separate one. Let's create a new group (folder) for these. Same for `test_table_drop.py`.